build.xml

```diff
@@ -1358,11 +1427,8 @@ ext = BeanUtils.cook(ext);
     <target name='server-war'>
         <ant dir='${server.dir}' target='war'/>
     </target>
-    <target name='taglib-jar'>
```

Comment:
Why do we need to remove this target `taglib-jar`? I also saw some taglib related changes above, do they really need here?

Thanks